tests/python/gaia-ui-tests/gaiatest/apps/base.py

```diff
@@ -13,14 +13,17 @@
 
 
 class Base(object):
+    DEFAULT_APP_HOSTNAME = '.gaiamobile.org'
+    DEFAULT_PROTOCOL = 'app://'
 
     def __init__(self, marionette):
         self.marionette = marionette
         self.apps = GaiaApps(self.marionette)
         self.accessibility = Accessibility(self.marionette)
         self.frame = None
-        self.manifest_url = hasattr(self, 'manifest_url') and self.manifest_url or None
         self.entry_point = hasattr(self, 'entry_point') and self.entry_point or None
+        self.DEFAULT_APP_HOSTNAME = 'gaiamobile.org'
```

Comment:
Please remove `self.DEFAULT_APP_HOSTNAME  = 'gaiamobile.org'` and `self.DEFAULT_PROTOCOL = 'app://'`. We don't need them.